### ISSUE TITLE
[Coachmark/TeachingBubble] Required ARIA attribute not present: aria-level

### ISSUE THREAD

**Issue by mario-msft** [microsoft/fluentui#16353]
### Environment Information

- **Package version(s)**: 7.123.1
- **Browser and OS versions**: Chrome v87.0.4280.88, Windows 10

### Describe the issue:

The `TeachingBubble` inside of a `Coachmark` is missing `aria-level` in the headline

#### Actual behavior:

No `aria-level` in headline
https://developer.microsoft.com/en-us/fluentui#/controls/web/coachmark

#### Expected behavior:

`aria-level` present in headline

### Documentation describing expected behavior

Running FastPass from Accessibility Insights for Web raised this error:
![image](https://user-images.githubusercontent.com/54645764/103569420-16c73f80-4e7c-11eb-9e78-f738bd92cf41.png)

**Comment by layershifter** [comment]
@mario-msft can you please link ARIA spec that says that `aria-level` is required?

I checked WAI-ARIA for [`role=heading`](https://www.w3.org/TR/wai-aria/#heading) and it says that `aria-level` defaults to `2`. The same thing is written on [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role#associated_wai-aria_roles_states_and_properties).

**Comment by mario-msft** [comment]
@layershifter Right, I saw that documentation too. Maybe it's a bug in Accessibility Insights for Web? The tool seems to require `aria-level`

**Comment by layershifter** [comment]
> @layershifter Right, I saw that documentation too. Maybe it's a bug in Accessibility Insights for Web? The tool seems to require `aria-level`

Yeah, I think so and suggest to report this issue to them 🐱
